i18n/en.json

```diff
@@ -482,6 +467,7 @@
   "admin.experimental.experimentalEnableDefaultChannelLeaveJoinMessages.title": "Enable Default Channel Leave/Join System Messages:",
   "admin.experimental.experimentalEnableHardenedMode.desc": "Enables a hardened mode for Mattermost that makes user experience trade-offs in the interest of security. See [documentation](!https://docs.mattermost.com/administration/config-settings.html#enable-hardened-mode-experimental) to learn more.",
   "admin.experimental.experimentalEnableHardenedMode.title": "Enable Hardened Mode:",
+  "admin.experimental.experimentalFeatures": "",
```

Comment:
Is this intentionally blank?

Reply:
Nope, good catch. The i18n string extraction tool wouldn't work directly in this case so I've had to manually search/replace those empty strings : )